src/modules/awake/Awake/Core/Manager.cs

```diff
@@ -294,19 +299,22 @@ internal static IntPtr GetHiddenWindow()
 
         public static void SetPassiveKeepAwakeMode(string moduleName)
         {
+            AwakeSettings currentSettings;
+
             try
             {
-                SettingsUtils settingsUtils = new SettingsUtils();
-                AwakeSettings settings = new AwakeSettings();
-
-                settings.Properties.Mode = AwakeMode.PASSIVE;
-                settingsUtils.SaveSettings(JsonSerializer.Serialize(settings), moduleName);
+                currentSettings = ModuleSettings!.GetSettings<AwakeSettings>(moduleName);
             }
-            catch (Exception ex)
+            catch (FileNotFoundException ex)
             {
                 string? errorString = $"Failed to reset Awake mode: {ex.Message}";
                 Logger.LogError(errorString);
+                currentSettings = new AwakeSettings();
             }
+
+            currentSettings.Properties.Mode = AwakeMode.PASSIVE;
+
+            ModuleSettings!.SaveSettings(JsonSerializer.Serialize(currentSettings), moduleName);
```

Comment:
What if we get an exception here too? Just feels like the code is more flaky with the changes.